### PR TITLE
docs(links): FRMW-272 refactor to atom

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-272

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `links` page to an atom.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `Links` Styleguide/Demo Page

screen shot 2016-02-26 at 2 32 53 pm

**Comment by 100stacks** [comment]
Initially added `QE` review.  Not needed since this is purely documentation.

**Comment by parlarjb** [comment]
LGTM

**Comment by ssalinas24** [comment]
LGTM
I understand that things are different from the mockups, and the key thing here was to get to the content in place. I know that @glynnis and @CITguy have already discussed these efforts.

**Comment by 100stacks** [comment]
@ssalinas24, sounds good.  @glynnis and @CITguy's conversation from PR https://github.com/rackerlabs/encore-ui/pull/1581 captured the future design/demo work  https://github.com/rackerlabs/encore-ui/pull/1581#issuecomment-188887856.

File: demo/templates/atoms/links.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <rx-page title="'Links'">
+  <hr />
```
Comment (CITguy) [inline demo/templates/atoms/links.html]: You can remove this horizontal rule. Cosmetic updates will happen in FRMW-711

File: demo/templates/atoms/links.html [hunk 2]
```diff
@@ -28,44 +29,50 @@ <h3 class="clear">Action Link Icons</h3>
...
-      When part of a list, they appear last.
-    </li>
-  </ul>
-  <rx-example name="link.colors"></rx-example>
+  <h4>Create/Add Actions</h4>
+  <p>
+    Green (<code>.msg-action</code>) is for create/add actions.
+    When part of a list, they appear at the top.
+  </p>
+  <rx-example name="link.colors.green.action"></rx-example>
```
Comment (CITguy) [inline demo/templates/atoms/links.html]: I'd exclude the color from the example file names. In this case, I'd name the example `link.colors.create`.

File: demo/templates/atoms/links.html [hunk 3]
```diff
@@ -28,44 +29,50 @@ <h3 class="clear">Action Link Icons</h3>
...
+  <h4>Download Actions</h4>
+  <p>
+    Blue (<code>.msg-info-blue</code>) is for download actions.
+    When part of a list, they appear second.
+  </p>
+  <rx-example name="link.colors.blue.info"></rx-example>
```
Comment (CITguy) [inline demo/templates/atoms/links.html]: I'd exclude the color from the example file names. In this case, I'd name the example `link.colors.download`.

File: demo/templates/atoms/links.html [hunk 4]
```diff
@@ -28,44 +29,50 @@ <h3 class="clear">Action Link Icons</h3>
...
+  <h4>Edit/Modify Actions</h4>
+  <p>
+    Orange (<code>.msg-info</code>) is for edit/modify actions.
+    When part of a list, they appear third.
+  </p>
+  <rx-example name="link.colors.orange.info"></rx-example>
```
Comment (CITguy) [inline demo/templates/atoms/links.html]: I'd exclude the color from the example file names. In this case, I'd name the example `link.colors.modify`.

File: demo/templates/atoms/links.html [hunk 5]
```diff
@@ -28,44 +29,50 @@ <h3 class="clear">Action Link Icons</h3>
...
+  <h4>Destructive Actions</h4>
+  <p>
+    Red (<code>.msg-warn</code>) is for destructive actions.
+    When part of a list, they appear last.
+  </p>
+  <rx-example name="link.colors.red.warning"></rx-example>
```
Comment (CITguy) [inline demo/templates/atoms/links.html]: I'd exclude the color from the example file names. In this case, I'd name the example `link.colors.destructive`.

File: demo/templates/atoms/links.html (hunk 2 above)
Comment (CITguy) [inline demo/templates/atoms/links.html]: What if we update to use a different color for action links? This separates the filename needing to know how the colors are implemented and allows us to retain commit history in the long haul (renaming resets history).

Comment (100stacks) [inline demo/templates/atoms/links.html]: @CITguy, sounds good.  I'm updating the example files now.

**Comment by 100stacks** [comment]
@CITguy, feedback implemented.  Thanks for the [Awesome Screenshot](https://chrome.google.com/webstore/detail/awesome-screenshot-screen/nlipoenfbbikpbjkfpfillcgkoblgpmj) protip for capturing longer webpages! :tada:

**Pull request merged**